modules/installation-rhv-downloading-ansible-playbooks.adoc

```diff
@@ -11,6 +11,18 @@ Download the Ansible playbooks for installing {product-title} version {product-v
 
 . On your installation machine, run the following commands:
 +
+ifdef::openshift-origin[]
+[source,terminal,subs=attributes+]
+----
+$ mkdir playbooks
+$ cd playbooks
+$ curl -s -L -X GET https://api.github.com/repos/openshift/installer/contents/upi/ovirt?ref=release-4.8  |
```

Comment:
This is master, so we need 4.10 I guess? Perhaps we should be using more generic "check `upi/ovirt` folder in github.com/openshift/install and pick a corresponding branch"?

Reply:
@vrutkovs Changed this to https://api.github.com/repos/openshift/installer/contents/upi/ovirt?ref=release-<version> ` with text to go to the openshift installer repo for the version.